README.md

```diff
@@ -10,7 +10,7 @@ To generate source code from the protobuf definitions contained in this project
 > **Note**: You must have [Buf CLI](https://docs.buf.build/installation) installed to run the following command.
 > 
 ```bash
-./buf.gen.yaml
+make buf-gen
```

Comment:
swapped this out to encourage people to use the make command so the hook is in place as early as possible